templates/basic_answerfile.txt.j2

```diff
@@ -0,0 +1,111 @@
+[environment:default]
+OVESETUP_DIALOG/confirmSettings=bool:True
+OVESETUP_CONFIG/applicationMode=str:both
+OVESETUP_CONFIG/remoteEngineSetupStyle=none:None
+OVESETUP_CONFIG/sanWipeAfterDelete=bool:False
+OVESETUP_CONFIG/storageIsLocal=bool:False
+OVESETUP_CONFIG/remoteEngineHostRootPassword=none:None
+{% if ovirt_engine_setup_firewall_manager %}
+OVESETUP_CONFIG/updateFirewall=bool:True
+OVESETUP_CONFIG/firewallManager=str:{{ ovirt_engine_setup_firewall_manager }}
+{% else %}
+OVESETUP_CONFIG/updateFirewall=bool:False
+OVESETUP_CONFIG/firewallManager=none:None
+{% endif %}
+OVESETUP_CONFIG/firewallChangesReview=none:None
+OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
+OVESETUP_CONFIG/fqdn=str:{{ ansible_fqdn }}
+OVESETUP_CONFIG/storageType=none:None
+{% if ovirt_engine_setup_require_rollback is defined %}
+OSETUP_RPMDISTRO/requireRollback=bool:{{ovirt_engine_setup_require_rollback}}
+{% else %}
+OSETUP_RPMDISTRO/requireRollback=none:None
+{% endif %}
+OSETUP_RPMDISTRO/enableUpgrade=bool:True
+OVESETUP_DB/secured=bool:False
+OVESETUP_DB/host=str:{{ovirt_engine_setup_db_host}}
+OVESETUP_DB/user=str:{{ovirt_engine_setup_db_user}}
+{% if ovirt_engine_setup_db_password is defined %}
+OVESETUP_DB/password=str:{{ovirt_engine_setup_db_password}}
+{% else %}
+OVESETUP_DB/password=none:None
+{% endif %}
+OVESETUP_DB/dumper=str:pg_custom
+OVESETUP_DB/database=str:{{ovirt_engine_setup_db_name}}
+OVESETUP_DB/fixDbViolations=none:None
+OVESETUP_DB/port=int:{{ovirt_engine_setup_db_port}}
+OVESETUP_DB/filter=none:None
+OVESETUP_DB/restoreJobs=int:2
+OVESETUP_DB/securedHostValidation=bool:False
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_DB/secured=bool:False
+OVESETUP_DWH_DB/host=str:{{ovirt_engine_setup_dwh_db_host}}
+OVESETUP_DWH_DB/user=str:{{ovirt_engine_setup_dwh_db_user}}
+OVESETUP_DWH_DB/database=str:{{ovirt_engine_setup_dwh_db_name}}
+OVESETUP_DWH_DB/port=int:{{ovirt_engine_setup_dwh_db_port}}
+{% else %}
+OVESETUP_DWH_DB/secured=none:None
+OVESETUP_DWH_DB/host=none:None
+OVESETUP_DWH_DB/user=none:None
+OVESETUP_DWH_DB/password=none:None
+OVESETUP_DWH_DB/database=none:None
+OVESETUP_DWH_DB/port=none:None
+{% endif %}
+{% if ovirt_engine_setup_dwh_db_password is defined %}
+OVESETUP_DWH_DB/password=str:{{ovirt_engine_setup_dwh_db_password}}
+{% else %}
+OVESETUP_DWH_DB/password=none:None
+{% endif %}
+OVESETUP_DWH_DB/dumper=str:pg_custom
+OVESETUP_DWH_DB/filter=none:None
+OVESETUP_DWH_DB/restoreJobs=int:2
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_DB/securedHostValidation=bool:False
+{% else %}
+OVESETUP_DWH_DB/securedHostValidation=none:None
+{% endif %}
+OVESETUP_ENGINE_CORE/enable=bool:True
+OVESETUP_CORE/engineStop=none:None
```

Comment:
Better drop this line, or set it to True. If you keep it, you rely on the fact that otopi lets the last occurrence of a key to override previous ones, and this isn't documented officially (so in theory might change in the future, or even make it alert/abort/whatever). This is because in upgrades you pass True. No harm in setting it here to True and remove from upgrades - clean setups will be ok.

Reply:
Thanks, I 'll try.